src/components/App/App.less

```diff
@@ -8,22 +8,31 @@
 @import "../TaskList/TaskList.less";
 @import "../TaskListItem/TaskListItem.less";
 @import "../Auth/Auth.less";
+@import "../TabbedNav/TabbedNav.less";
 
 body {
-  background-color: #fff;
+  background-color: white;
   min-height: 100vh;
 }
 
 .button.is-primary {
   margin-right: 5px;
 }
 
-.top-nav {
+.top-nav, .secondary-nav {
   z-index: 5;
   margin-bottom: 15px;
   border-bottom: 1px solid rgba(211, 214, 219, 0.5);
 
-  .button.is-link {
+  .button.is-link, .label {
     text-decoration: none;
   }
 }
+
+nav#tabbed-nav {
+  margin-bottom: 0 !important;
```

Comment:
Yes, this was to work around Bulma. I may be able to remove this, I think I found a specific enough selector to not need it anymore.